Terminal.Gui/Core/View.cs

```diff
@@ -124,27 +124,27 @@ internal enum Direction {
 		/// <summary>
 		/// Event fired when the view gets focus.
 		/// </summary>
-		public event EventHandler<FocusEventArgs> Enter;
+		public Action<FocusEventArgs> Enter;
 
 		/// <summary>
 		/// Event fired when the view looses focus.
 		/// </summary>
-		public event EventHandler<FocusEventArgs> Leave;
+		public Action<FocusEventArgs> Leave;
 
 		/// <summary>
 		/// Event fired when the view receives the mouse event for the first time.
 		/// </summary>
-		public event EventHandler<MouseEventEventArgs> MouseEnter;
+		public Action<MouseEventEventArgs> MouseEnter;
```

Comment:
Is it just me, or does `MouseEventEventArgs` seem silly? Should we just make these `MouseEventArgs`?

Reply:
BTW, I know I'm the one who did this ;-)

Reply:
`MouseEventArgs `really looks much better :-)